docs/getting-started-guides/ubuntu/installation.md

```diff
@@ -172,16 +172,15 @@ Copy the kubeconfig file to the default location.
 juju scp kubernetes-master/0:config ~/.kube/config
 ```
 
-Fetch a binary for the architecture you have deployed. If your client is a
-different architecture you will need to get the appropriate `kubectl` binary
-through other means. In this example we copy kubectl to `~/bin` for convenience,
-by default this should be in your $PATH.
+The next step is to install the kubectl client on your local machine. The recommended way to do this on Ubuntu is using the kubectl snap ([https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-snap-on-ubuntu](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-snap-on-ubuntu)):
 
 ```
-mkdir -p ~/bin
-juju scp kubernetes-master/0:kubectl ~/bin/kubectl
+# Run this on machine you wish to use to control the kubernetes cluster.
```

Comment:
Replace with:
```
# Run this on the machine you want to use to control the kubernetes cluster.
```

Reply:
@zacharysarah I'm confused, what exactly do you want me to change to fix this?

Reply:
@zacharysarah sorry to bump, but what exactly do you want me to change here?

Reply:
@calvinhartwell There’s a missing article. If you replace the entire line with the line I provided, that should take care of it.